sw/device/mask_rom/boot.md

```diff
@@ -27,62 +28,76 @@ Sub-procedures are documented below the main `boot` function.
 
 ```c
 void boot(void) {
-  boot_reason = read_boot_reason();
+  boot_reason = read_boot_reason(); // Boot Policy Module
 
   // Clean Device State Part 2.
   // See "Cleaning Device State" Below.
-  clean_device_state_part_2(boot_reason);
+  clean_device_state_part_2(boot_reason); // Chip-Specific Startup Module
 
   // Enable Memory Protection
   // - PMP Initial Region (if not done in power on)
-  enable_memory_protection();
+  enable_memory_protection();  // Lockdown Module
 
   // Chip-specific startup functionality
   // **Open Q:** Proprietary Software Strategy.
   // - Clocks
   // - AST / Entropy.
   // - Flash
   // - Fuses
-  chip_specific_startup();
+  chip_specific_startup(); // Chip-Specific Startup Module
```

Comment:
I think this also deserves a section in "Subroutines", or, now that we have the "Modules" section, maybe the sections under "Subroutines" can move there.

Reply:
Yes, I need to move around subroutines so they're each grouped by module, but probably not in this PR.